src/inmanta/moduletool.py

```diff
@@ -620,25 +620,6 @@ def modules_parser_config(cls, parser: ArgumentParser, parent_parsers: abc.Seque
             parents=parent_parsers,
         )
 
-        commit = subparser.add_parser("commit", help="Commit all changes in the current module.")
```

Comment:
This is not sufficient, you need to remove the `commit` method as well (and any dead code it causes)